data-new/CultureAndOrganization/EducationAndGuidance.yaml

```diff
@@ -114,9 +113,9 @@ Education and Guidance:
       resources: 1
     usefulness: 3
     level: 3
-    samm: IM-3, ST-3, SR2-B
-    iso27001-2017:
-    - 16.1.6
+    references:
+    - samm:IM-3, ST-3, SR2-B
```

Comment:
Should this become 
samm:IM-3
samm:ST-3
samm:SR2-B
?

Reply:
yes!